src/utils/commandRunner.ts

```diff
@@ -31,7 +31,7 @@ export class CommandRunner {
   }> {
     let executablePath: string;
     let command: string;
-    let runEnv: NodeJS.ProcessEnv | undefined;
+    let runEnv: NodeJS.ProcessEnv = {};
```

Comment:
`ProcessEnv` extends Dict type so the line can be replaced as `let runEnv: NodeJS.ProcessEnv`
```
            // Alias for compatibility
            interface ProcessEnv extends Dict<string> {}
```

Reply:
Typescript is smart enough to know that an empty dictionary does respect the `Dict<string>` so I was able to do that without extra changes.